src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.c

```diff
@@ -202,6 +217,12 @@ static void wrapped_rr_closure(grpc_exec_ctx *exec_ctx, void *arg,
                 (void *)*wc_arg->target, (void *)wc_arg->rr_policy);
         abort();
       }
+      // Pass on client stats via context. Passes ownership of the reference.
```

Comment:
Is there any real value in this consistency?  I don't know of any style rule that says that we have to use the same style comments throughout a given file.  I personally prefer C++-style comments for full-line or end-of-line comments, although I do use C-style comments for in-line comments (e.g., to document the name of each parameter for a function call with many parameters).  I don't really see a problem with mixing the two in the same file.